package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Web application (server + web client) for VelociRaffle",
   "main": "index.js",
   "scripts": {
+    "build": "babel server -d serverDist --presets es2015",
     "precommit": "jscs . --fix && eslint ./client/app && npm test",
+    "postinstall": "npm run build",
     "dev-client": "gulp",
-    "dev-server": "nodemon server.js",
+    "dev-server": "nodemon server/index.js --exec babel-node --presets es2015",
```

Comment:
:+1: Nice. I assume this is how you can run the most recent version of ES in Node?